Sources/Brave/Extensions/UIViewExtensions.swift

```diff
@@ -14,6 +14,12 @@ extension UIView {
 
     let offset = offset ?? .zero
 
+    // Temporary check to handle _UIGraphicsBeginImageContextWithOptions zero size error
+    // Shiould be replaced with UIGraphicsImageRenderer
+    if size.width <= 0 || size.height <= 0 {
```

Comment:
Maybe replace with guard to make it easier to read?
And `UIGraphicsBeginImageContextWithOptions` is used in 2 other places in code what about adding guards to it as well?

Reply:
Fix at https://github.com/brave/brave-ios/pull/8269/commits/c3e84bf9508358e759380ac61920630996837e8b  Multiple usages handled and guard is used for readability issue